ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/SigningEeaSendTransactionIntegrationTest.java

```diff
@@ -433,4 +437,21 @@ void invalidParamsResponseWhenMissingRestriction() {
         request.ethSigner(sendTransaction.missingRestriction()),
         response.ethSigner(INVALID_PARAMS));
   }
+
+  @Test
+  void missingNonceResultsInRequestToPrivGetEeaTransactionCount() {
+    final String ethNodeResponseBody = "VALID_RESPONSE";
+    final String requestBody = sendRawTransaction.request(sendTransaction.withNonce("0x1"));
+
+    setUpEthNodeResponse(request.ethNode(requestBody), response.ethNode(ethNodeResponseBody));
+
+    sendRequestThenVerifyResponse(
+        request.ethSigner(sendTransaction.missingNonce()), response.ethSigner(ethNodeResponseBody));
+
+    final HttpRequest[] httpRequests = clientAndServer.retrieveRecordedRequests(null);
```

Comment:
Rather the verify method was in the IntegrationTestBase class as this forms part of the DSL. Also this could just use the verify method on the mock server. It can do pattern matching if that's what is needed.